cmd/controller-manager/app/controllers.go

```diff
@@ -348,6 +352,38 @@ func addAllControllers(mgr manager.Manager, client k8s.Client, informerFactory i
 		addController(mgr, "volumesnapshot", volumeSnapshotController)
 	}
 
+	// "pvc-autoresizer"
+	monitoringOptionsEnable := cmOptions.MonitoringOptions != nil && len(cmOptions.MonitoringOptions.Endpoint) != 0
+	if monitoringOptionsEnable {
+		if cmOptions.IsControllerEnabled("pvc-autoresizer") {
+			if err := runners.SetupIndexer(mgr, false); err != nil {
+				return err
+			}
+			promClient, err := runners.NewPrometheusClient(cmOptions.MonitoringOptions.Endpoint)
+			if err != nil {
+				return err
+			}
+			pvcAutoResizerController := runners.NewPVCAutoresizer(
+				promClient,
+				mgr.GetClient(),
+				ctrl.Log.WithName("pvc-autoresizer"),
+				1*time.Minute,
+				mgr.GetEventRecorderFor("pvc-autoresizer"),
+			)
+			addController(mgr, "pvcautoresizer", pvcAutoResizerController)
+		}
+	}
+
+	if cmOptions.IsControllerEnabled("workload-Restart") {
```

Comment:
`workload-Restart` is a bad controller name, and its style is inconsistent, maybe choose another name.

Reply:
How about `expansion-restarter` or`autorestarter`?  Or any other better suggestions?

Reply:
yeah, sounds better. This controller is only used when PVC size changes, right? So, how about `pvc-workload-restarter`?

Reply:
Yes it does, `pvc-workload-restarter` sounds great, I'll update it, and thanks for the suggestion.